docs/faq.mdx

```diff
@@ -0,0 +1,26 @@
+---
+slug: /faq
+displayed_sidebar: mainSidebar
+---
+
+# FAQ
+
+Below is a list of the frequently asked questions we have encountered from users. If you have a question and you don't see it addressed here, please let us know by [sending us a message](./support).
+
+## General
+
+### I was using the older version of Superface, what should I do?
+
+If you were using the previous version of the Superface app, and using our pre-made Comlinks with OneSDK and the CLI, you can continue to do so for the time being. However, we do plan to deprecate the hosted Comlink functionality later this year so we recommend that you move to the latest version of OneSDK sooner rather than later. Find out more about our [suggested approach](./migrate.mdx).
+
+### Is Superface ready for production use?
+
+Superface's OneSDK is currently in beta.
+
+## CLI
+
+Specific questions related to the Superface CLI.
+
+### What does `Unexpected status code 401 received` mean?
+
+This error means that the CLI cannot communicate with the Superface servers because it is not logged in. To rectify this, run `superface login` and follow the on screen commands.
```

Comment:
FYI this [was addressed in the CLI](https://github.com/superfaceai/cli/pull/308) already